### PR TITLE
FOR REVIEW: Documenting processes + checklists

### PR DESCRIPTION
**Pull request by LongForWisdom**

### CODE REVIEW

File: processes/executive-copy-creation-checklist.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+
+# Executive Creation
+
+## Creation
+
+- [ ] Sync Community Git
+- [ ] Ensure you are working from the latest master branch
+- [ ] Read Associated Executive Contents sheet
```
Comment (patrick-j-govalpha) [inline processes/executive-copy-creation-checklist.md:8]: add link to sheet?

File: processes/executive-copy-creation-checklist.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+- [ ] Ensure that the office-hours modifier is mentioned as part of the executive copy, if present in the executive spell code.
+- [ ] Ensure that it is clearly stated which items are affected by the GSM Delay.
+- [ ] Ensure that the GSM Delay period is clearly communicated.
+- [ ] Ensure that the proposal expiry is explicitly noted, including the length of time for which the proposal is valid.
+
+## Items
+
+- [ ] Add each item to the proposal details section
+- [ ] Order the items by judged impact/importance
```
Comment (patrick-j-govalpha) [inline processes/executive-copy-creation-checklist.md:28]: per crafters request - try and make sure that the order in the sheet and the order in the copy match to aid code review

File: processes/executive-copy-creation-checklist.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+## Self-Review
+
+- [ ] Do any '$' parameter signifiers remain? (spell address is unknown at this point)
+- [ ] Do the executive title, summary, and executive summary accurately reflect executive copy content?
+- [ ] Read the executive copy in full, with the above checklists and general readability in mind. Make any required changes.
+- [ ] Run the copy through a spelling and grammar checker - do not blindly accept suggested changes.
+- [ ] Inform the reviewer that the executive copy is ready to review
+- [ ] Commit changes, and create a PR to the Community GitHub repository.
+
+## Submission
```
Comment (patrick-j-govalpha) [inline processes/executive-copy-creation-checklist.md:50]: add provide copy to growth team for twitter?

File: processes/poll-submission-checklist.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+Complete once.
+
+- [ ] Check that all the polls you expected to be active are active.
+- [ ] Communicate weekly polls to the #governance channel in the official MakerDAO discord.
+- [ ] Update [Governance Tracker sheet](https://docs.google.com/spreadsheets/d/1LWNlv6hr8oXebk8rvXZBPRVDjN-3OrzI0IgLwBVk0vM/edit#gid=0) with new live polls
+
+
+
+
```
Comment (patrick-j-govalpha) [inline processes/poll-submission-checklist.md:58]:
```suggestion
```

**Review by patrick-j-govalpha — COMMENTED** [review]
left some comments, on the whole this is a very good start

File: processes/executive-copy-creation-checklist.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+- [ ] Update dates - exact time is not required and should be left at midnight.
+
+- [ ] Ensure the title refers to the most impactful/important actions included in the proposal.
+- [ ] Ensure that the summary refers to all actions included in the proposal.
+
+- [ ] Ensure that the publisher of the executive is listed in the preamble (at the entity level).
+- [ ] Ensure that the writer of the executive is listed in the preamble (at the entity level).
+- [ ] Ensure that the reviewers of the executive are listed in the preamble (at the entity level).
+
+- [ ] Ensure that the office-hours modifier is mentioned as part of the executive copy, if present in the executive spell code.
```
[the remark this answers is not in the document]
Comment (LongForWisdom) [reply processes/executive-copy-creation-checklist.md:20]: This document feels like the wrong place to do it. Not sure where is the right place though, will have a think.

File: processes/executive-copy-creation-checklist.md (hunk 3 above)
Comment (LongForWisdom) [reply processes/executive-copy-creation-checklist.md:50]: I could do, this checklist basically already exists in the revamped executive sheet. That includes all the external wants. I'm tempted to either remove this checklist from here, or just make it clear that these are the 'internal' requirements, and there are other external things to do.

**Review by patrick-j-govalpha — APPROVED** [review]

**Pull request merged**